src/components/notes/note-list.tsx

```
@@ -0,0 +1,119 @@
+  void handleDelete; // suppress unused warning — used in future Phase 4
+
+  return (
+    <div className="flex flex-1 flex-col">
+      <div className="flex items-center justify-between border-b p-3">
+        <h2 className="text-sm font-semibold text-gray-700">Notes</h2>
+        <button
+          onClick={onCreateNote}
+          className="rounded p-1 text-gray-500 hover:bg-gray-200 hover:text-gray-700"
+          aria-label="New note"
+        >
+          <svg className="h-4 w-4" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)